### ISSUE TITLE
Fix WebUSB / BOS descriptors

### ISSUE THREAD

**Issue by xobs** [im-tomu/tomu-bootloader#16]
The BOS descriptors are reporting issues on Chrome 68.  These should be fixed.

**Comment by devanlai** [comment]
I'm not sure about the BOS descriptor - at a glance, the code looks fine.
For WebUSB, the landing page URL descriptor is retrieved via a vendor request, not a standard `GET_DESCRIPTOR` request. It should look similar to the WinUSB handler, something like:

```
    case (WEBUSB_VENDOR_CODE << 8) | 0xC0:
        if (dev->dev_req.wIndex == WEBUSB_REQ_GET_URL)
        {
            if (dev->dev_req.wValue == 0x0001)
            {
                // Return the landing page URL descriptor
                data = (const uint8_t*)&landing_url_descriptor;
                datalen = LANDING_PAGE_DESCRIPTOR_SIZE;
                break;
            }
        }
        usb_lld_ctrl_error(dev);
        return;
```

**Comment by xobs** [comment]
Closed by #35
